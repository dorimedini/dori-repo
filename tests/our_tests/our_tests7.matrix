{
matrix m1(1,3) = [1,2,3];
matrix m2(3,3) = transpose(m1)*m1;
matrix m3(1,1) = m1*transpose(m1);
m1 = -2*m1;
matrix m4(1,3) = [4,5,6];
m2 = transpose(m1)*m4;
print m2;
foreach (m1 in m2) {
	print m1;
}

}