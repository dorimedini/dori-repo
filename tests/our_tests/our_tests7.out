[-8,-10,-12;-16,-20,-24;-24,-30,-36]
-8
-10
-12
-16
-20
-24
-24
-30
-36
